### ISSUE TITLE
cannot access other flatpak programs data from the flatpak release - permissions issue?

### ISSUE THREAD

**Issue by BrainDamage** [zocker-160/SyncThingy#20]
I'd like to use syncthing to syncronize my games' save progress between multiple machines using the flatpak release of syncthingy; the problem is that they are flatpak programs on their own. I attempt to add eg ~/.var/app/packagename/data/ to the syncronized paths, except that syncthing reports the path as empty.
If I bind-mount the paths externally they become visible and writeable, but this is extremely awkward and impractical to use.
I assume this is all because syncthingy has no permissions to read/write into other flatpak path programs?

**Comment by zocker-160** [comment]
Yes this is a permission issue on the Flatpak side. Can you try adding `~/.var/app` and `xdg-data/flatpak` to the filesystem permissions using Flatseal?

Don't forget to restart Synthingy to make the changes apply.

**Comment by BrainDamage** [comment]
Adding the permissions using flatseal fixed the issue. Thanks!
